### PR TITLE
Gacd inline content definition

### PR DESCRIPTION
**Pull request by IvanBorislavovDimitrov**
LMCROSSITXSADEPLOY-1857

### CODE REVIEW

File: com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/Constants.java [hunk 1]
```diff
@@ -102,7 +102,6 @@
     public static final String VAR_ALL_MODULES_TO_DEPLOY = "allModulesToDeploy";
     public static final String VAR_APPS_TO_DEPLOY = "appsToDeploy";
     // TODO: Do we use this variable?
```
Comment (radito3) [inline com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/Constants.java]: Delete this comment as well

File: com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/FileUtils.java [hunk 2]
```diff
@@ -57,6 +62,22 @@ public static boolean isDirectory(String fileName) {
         return fileName.endsWith("/");
     }
 
+    public static void cleanUp(Path filePath, Logger logger) {
```
Comment (radito3) [inline com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/FileUtils.java]: Didn't we have this function elsewhere?

File: com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/UploadAppStep.java [hunk 3]
```diff
@@ -185,10 +188,10 @@ private void cleanUp(Path filePath) {
         }
 
         try {
-            getStepLogger().debug(Messages.DELETING_TEMP_FILE, filePath);
+            getStepLogger().debug(DELETING_TEMP_FILE, filePath);
```
Comment (radito3) [inline com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/UploadAppStep.java]: Why the change to static imports?

File: com.sap.cloud.lm.sl.cf.web/src/test/java/com/sap/cloud/lm/sl/cf/web/api/impl/OperationsApiServiceImplTest.java [hunk 4]
```diff
@@ -191,7 +194,12 @@ public void testExecuteOperationActionUnauthorized() {
 
     @Test
     public void testStartOperation() {
-        testedClass.startOperation(mockHttpServletRequest(EXAMPLE_USER), SPACE_GUID, createOperation(null, null));
+        Map<String, Object> parameters = new HashMap<>();
+        parameters.put(Constants.PARAM_MTA_ID, "test");
+        Operation operation = createOperation(null, null, parameters);
```
Comment (radito3) [inline com.sap.cloud.lm.sl.cf.web/src/test/java/com/sap/cloud/lm/sl/cf/web/api/impl/OperationsApiServiceImplTest.java]: If the parameters is only going to be a one-entry map, you could use MapUtil.asMap

File: pom.xml [hunk 5]
```diff
@@ -63,7 +63,7 @@
         <guava.version>23.6.1-jre</guava.version>
         <jsr305.version>3.0.2</jsr305.version>
         <immutables.version>2.8.3</immutables.version>
-        <com.sap.cloud.lm.sl.open.version>1.89.0</com.sap.cloud.lm.sl.open.version>
+        <com.sap.cloud.lm.sl.open.version>1.90.0-SNAPSHOT</com.sap.cloud.lm.sl.open.version>
```
Comment (radito3) [inline pom.xml]: Wouldn't this be better off in its own commit?

File: com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/FileUtils.java (hunk 2 above)
Comment (IvanBorislavovDimitrov) [reply com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/FileUtils.java]: Yes, I moved it here because it was copy-pasted.

File: com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/Constants.java (hunk 1 above)
Comment (IvanBorislavovDimitrov) [reply com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/Constants.java]: There are plenty of these comments and I think they are related to "Variables" change.

File: com.sap.cloud.lm.sl.cf.web/src/test/java/com/sap/cloud/lm/sl/cf/web/api/impl/OperationsApiServiceImplTest.java (hunk 4 above)
Comment (IvanBorislavovDimitrov) [reply com.sap.cloud.lm.sl.cf.web/src/test/java/com/sap/cloud/lm/sl/cf/web/api/impl/OperationsApiServiceImplTest.java]: Done

File: pom.xml (hunk 5 above)
Comment (IvanBorislavovDimitrov) [reply pom.xml]: Done
